### ISSUE TITLE
Is it possible to use React or VueJS ?

### ISSUE THREAD

**Issue by muhajirframe** [Gottwik/Enduro#43]
Hi @Gottwik, pretty cool project. Love it. This is exactly what i was looking for, static site generator with an admin interface :+1: .

Is it possible to use React or VueJs with Enduro?
I mean there's a pretty cool project too called [nuxtjs](https://nuxtjs.org) , basicly it can generate static version of vuejs, and there's also an with expressjs (https://github.com/nuxt/express).

I am just wondering is it possible to use the power of enduro together with nuxtjs?

And also if i use Enduro with static serving. Can i upload the built files to netlify and still have access to admin interfaces? 

Thank you

**Comment by Gottwik** [comment]
Well, good quetion, but I am not able to answer it since I never tried react nor vuejs. I suggest you go ahead and try it, possibly other people could benefit from it as well. What would enduro.js need tu support to work with react and vuejs?

Netlify works great though, there is even an doc article about it: http://www.endurojs.com/docs/netlify

What you don't get on netlify is the admin interface(requires logic to log in and change file.).

**Comment by CosmoMyzrailGorynych** [comment]
TL;DR: Yes.

Both react and Vue.js, and even riot can be easily compiled with gulp and used on your client pages. You cannot use it in admin interface, though, because it uses angular js.

**Comment by CosmoMyzrailGorynych** [comment]
There is no place to plug in compilers or static renderers for now, including nuxtjs; it is something you need to implement by yourself.

**Comment by archcra** [comment]
<img width="356" alt="screen shot 2017-07-12 at 3 51 56 pm" src="https://user-images.githubusercontent.com/6466550/28107421-4acbd16c-671a-11e7-9500-aeb7b8101150.png">

The link is broken.

**Comment by archcra** [comment]
I hope there is a demo of integrated with vue.js

**Comment by archcra** [comment]
I'm  now using vue.js page by page (in each .hbs).
And, so far so good.

**Comment by Gottwik** [comment]
Awesome to hear 😄 . I'll close this ticket as resolved, feel free to open if anybody have further questions on this.
